haaska.py

```diff
@@ -144,6 +144,8 @@ def mk_appliance(x):
             o['friendlyName'] = x['attributes']['friendly_name']
             if entity_domain(x) == 'scene':
                 o['friendlyName'] += ' Scene'
+            elif entity_domain(x) == 'group':
```

Comment:
I'm not a big fan of it either, but I just relied on `haaska_name` in `customize` to make it even more natural.